src/core/agent/services/multiSigService.ts

```diff
@@ -568,9 +589,9 @@ class MultiSigService extends AgentService {
       recp,
       {
         gid: serder.pre,
-        smids: rmids,
-        rmids: rmids,
-        rstates: states,
+        smids: smidsMember,
+        rmids: rmidsMember,
```

Comment:
Hmm, `smids` and `rmids` before were the list of identifiers, as they should be. You have converted them to objects with the key state.

Please read my original comment on this PR:
```
    const smids = members?.signing;
    const rmids = members?.rotation;
```
Just pass these to here. And that's it.

`recp` can contain all unique values of `smids` and `rmids`.

Reply:
@iFergal 
Currently, `smidsMember` is an array storing the values of `members?.signing` after `signifyClient.keyStates().query(signing.aid, nextSequence);`, similar to `rmidsMember`.

So, do we need to continue doing this?

And the values of states and rstates in:
```
const icp = await this.props.signifyClient
  .identifiers()
  .rotate(name, { states: smids, rstates: rmids });
```
will be smids: members?.signing and rmids: members?.rotation, right?

Reply:
Good point, we need `states` and `rstates` too for this line. So no, these need to have the `keyStates().query` results. But for `smids` and `rmids` in the exn message, we don't need the unnecessary conversions from `smidsMember` back to `smids` you have.

Change `rotateMultisig` to accept `smids`, `rmids`, `states` and `rstates`.

`states` is `smidsMember` (please rename it to states, as smidsMember is incorrect naming), `rstates` is currently what you have named `rmidsMember`